components/Flipper/Flipper.tsx

```diff
@@ -407,6 +407,48 @@ function Flipper(props: Props) {
         )
     }
 
+    function getCardTitleElement() {
+        if (isLoading) {
+            return getLoadingElement(<p>Logging in with Google...</p>)
+        }
+        if (!isLoggedIn) {
+            return <h2>Free Auction Flipper</h2>
+        }
+        if (hasPremium) {
+            let type = getPremiumType(activePremiumProduct)
+            if (type.priority === PREMIUM_RANK.STARTER) {
+                return (
+                    <span>
+                        You are using starter premium. Check details <Link href="/premium">here</Link>
+                    </span>
+                )
+            }
+            if (type.priority === PREMIUM_RANK.PREMIUM) {
+                return (
+                    <span>
+                        You are using premium. Check details <Link href="/premium">here</Link>
+                    </span>
+                )
+            }
+            if (type.priority === PREMIUM_RANK.PREMIUM_PLUS) {
+                return (
+                    <span>
+                        You are using premium+. Check details <Link href="/premium">here</Link>
```

Comment:
please make an element to display the accountTier and use it everywhere. on the account page it displays it incorrectly for example 
![image](https://user-images.githubusercontent.com/16632490/187764961-a3a966ca-986c-4e77-9c3d-396e8a2d9f30.png)